docs/root/version_history/current.rst

```diff
@@ -33,6 +33,9 @@ Minor Behavior Changes
   instead of `//source/common/chromium_url`. The new path canonicalizer is enabled by default. To
   revert to the legacy path canonicalizer, enable the runtime flag
   `envoy.reloadable_features.remove_forked_chromium_url`.
+* http: increase the maximum allowed number of initial connection WINDOW_UPDATE frames sent by the peer from 1 to 5.
+* http: upstream flood and abuse checks increment the count of opened HTTP/2 streams when Envoy sends initial HEADERS frame for the new stream. Before the counter was incrementred when
```

Comment:
nit: Long line with line-break.  Either merge the next line into this one, or apply line breaks around 100 columns instead of 185.

Reply:
done